plugins/bbcode/plugin.js

```diff
@@ -3,6 +3,83 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+/**
+ *		// This is actually the default value.
```

Comment:
It would be nice to provide some readable docs for exposed config options. Also, there is probably no need to attach a copy of the config group as a comment, it's already available via source button.